package.json

```diff
@@ -5,8 +5,8 @@
   "author": "Timur Shemsedinov <timur.shemsedinov@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "test": "eslint ./Exercises; hpw",
-    "ci": "eslint ./Exercises && hpw"
+    "test": "eslint .\\Exercises && hpw",
+    "ci": "eslint .\\Exercises && hpw"
```

Comment:
Don't change or don't commit if changed

Reply:
Thank you very much for your time. This pull request was created by mistake. My intention was to test stuff within my fork and not to bother anyone. My Apologies

Reply:
That's fine)